proxy/api/src/main.rs

```diff
@@ -86,9 +87,38 @@ async fn run(
         temp: _dont_drop_me,
         ctx,
         peer,
+        seeds_sender,
         subscriptions,
     } = rigging;
 
+    if let Some(seeds_sender) = seeds_sender {
+        let seeds_store = ctx.store.clone();
+        tokio::spawn(async move {
+            let mut last_seeds: Vec<seed::Seed> = vec![];
+            let mut timer = tokio::time::interval(Duration::from_secs(1));
+
+            loop {
+                let _timestamp = timer.tick().await;
+
+                let seeds = session::settings(&seeds_store).await.unwrap().coco.seeds;
+                let seeds = seed::resolve(&seeds).await.unwrap_or_else(|err| {
+                    log::error!("Error parsing seed list {:?}: {}", seeds, err);
+                    vec![]
+                });
+
+                if seeds == last_seeds {
+                    continue;
+                }
+
+                if seeds_sender.broadcast(seeds.clone()).is_err() {
```

Comment:
Would be helpful to `log.error!` here, no?

Reply:
The error is not really helpful as it means all receivers are gone. It determines that this task should stop continuing its work. While we suffer from a complex graph of long-running routines for now, we should be most adamant about releasing resources when shutting down/reloading. Proxy's main binary is overdue for a more structured approach to plumbing taking the `Rigging` idea to the next level.